app/charts/shared/chart-helpers.spec.tsx

```diff
@@ -231,4 +232,16 @@ describe("getChartConfigComponentIris", () => {
       "https://environment.ld.admin.ch/foen/nfi/unitOfReference",
     ]);
   });
+
+  it("should return correct componentIris for dual line chart (join by)", () => {
+    const componentIris = extractChartConfigComponentIris(
+      dualLine1Fixture as unknown as ChartConfig
+    );
+    expect(
+      componentIris.includes(
+        "https://energy.ld.admin.ch/sfoe/bfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen/Jahr"
+      )
+    ).toBe(true);
+    expect(componentIris.includes("joinBy__0")).toBe(false);
```

Comment:
Nit: I wonder if we could a helper function here to create this iri? I think I've seen some function that took an index and returned this string 👀

Reply:
👍 Done.